### PR TITLE
docs(readme): Add new related project, lighthouse-github-reporter

### PR DESCRIPTION
**Pull request by carlesnunez**
**Summary**
This PR ads a new related project into Lighthouse called Lighthouse-github-reporter. More info on the issue.
**Related Issues/PRs**
https://github.com/GoogleChrome/lighthouse/issues/6097

### CODE REVIEW

**Comment by carlesnunez** [comment]
I signed it!

File: readme.md [hunk 1]
```diff
@@ -276,6 +276,8 @@ This section details services that have integrated Lighthouse data. If you're wo
 ## Related Projects
 Other awesome open source projects that use Lighthouse.
 
+* **[lighthouse-gh-reporter](https://github.com/addyosmani/webpack-lighthouse-plugin)** - give Lighthouse-CI the power of put reports as a comment on your pull requests.
```
Comment (wardpeet) [inline readme.md]: I believe your url is wrong. It's the same url as webpack-lighthouse-plugin

Comment (exterkamp) [inline readme.md]: Checking the history of this file, we add new integrations to the bottom of the list usually.  So I'd move this to right under lighthouse4u.

Comment (exterkamp) [reply readme.md]: nit: give Lighthouse-CI the power ~~of put~~ _to make_ reports as a comment on your pull requests _in Github_.

**Comment by carlesnunez** [comment]
PR Changes done! Sorry for the typo errors and position mistake! 👍

**Comment by carlesnunez** [comment]
Hello! Why is the CI failing exactly? What I did wrong?

**Comment by wardpeet** [comment]
It's appveyor, no worries it's pretty flaky

**Comment by carlesnunez** [comment]
Thank you for your word fixes @brendankenny ! 😸

**Comment by brendankenny** [comment]
@wardpeet if you're good here

**Pull request merged**